src/app/modules/category/views/performance/performance.component.ts

```diff
@@ -83,6 +78,12 @@ export class CategoryPerformanceComponent extends CategoryBase implements OnDest
   public _showExternalLink = true;
   public _showCustomLegend = false;
 
+  public _tableModesOptions = [
+    ...(this._viewConfig.userFilter !== null ? { label: this._translate.instant('app.engagement.dimensions.users'), value: TableModes.users } : []),
```

Comment:
@adimachmali note that if you hide the user filter - you still see the users table since it is the first option. You need to handle the table view as well.

Reply:
fixed.